.github/workflows/ci.yml

```diff
@@ -9,8 +9,8 @@ jobs:
         shell: bash
         run: |
            sudo apt-get update
-           sudo apt-get install bats bash
```

Comment:
Do we need this installed twice?

argcomplete

Reply:
Nope, I think the issue I am seeing is because argcomplete is not in the Container.